enhancements/ingress/custom-route-configuration.md

```diff
@@ -193,6 +194,23 @@ pairs.
 Library-go is also an ideal spot to provide functionality to inject route hosts.  Something like our StaticResource
 controller which automatically handles the API status and the required route changes.
 
+#### Trusting the oauth-server
+
+Some OpenShift components need to communicate with the OpenShift oauth-server in order to
+retrieve access tokens on behalf of the user that wants to use them. Typical examples include
+the web console and anything that uses the oauth-proxy, like the monitoring tools. A custom oauth-server
+serving certificate might break trust from the other components in the cluster.
+
+The authentication operator needs to publish the oauth-server's  trust root in a well-known location so
```

Comment:
you don't always have the trust.  You sometimes only have the serving cert.  The golang TLS negotiation doesn't require a chain back to the root.

Reply:
ack, I think this is a leftover from a previous version.